CHANGES.md

```diff
@@ -1,3 +1,20 @@
+## edge-19.10.4
+
+This edge release adds dashboard UX enhancements, Open Census tracing
+for the control plane components, and improves the speed of the CLI.
+
+* CLI
+  * Made `linkerd install --ignore-cluster` and `--skip-checks` faster
+  * Fixed a bug causing `linkerd upgrade` to fail when used with 
+  `--from-manifest`
+* Controller
+  * Added `--trace-collector` and `--sampling-probability` flags to the 
```

Comment:
The configuration for this hasn't merged yet so I'd omit this from the change notes.